tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/ProcessStart/ProcessStartIntegration.cs

```diff
@@ -0,0 +1,59 @@
+// <copyright file="ProcessStartIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.ComponentModel;
+using Datadog.Trace.ClrProfiler.CallTarget;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Process
```

Comment:
The namespace `Process` doesn't match the folder name `ProcessStart`, so I'd like to choose a single name. In a previous comment I suggested the idea of naming this integration `Process`  (though the name of this class is fine because it targets a specific method). What do you think?

Reply:
I think that renaming the folder to process would be the best choice. I will go for that.